### PR TITLE
conforms readme to template, adds more detail on what error handling is

### PR DESCRIPTION
**Pull request by tildeee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,39 +13,70 @@
...
-## Baseline
-You should write your program as a series of statements within a file `calculator.rb`. Anyone with this file should be able to run the command `$ ruby calculator.rb` to run the program. The program should ask the user for an operation (as a string symbol) and two numbers.
+## Objective
+
+We will build a calculator command line interface (CLI) that allows any user to interact with the command line to perform simple arithmetic. The program should ask the user to type in and enter in pieces of input. Based on the input that the user typed in, the program should output its evaluation.
+
+## Getting Started
+
+You should write your program as a series of statements within a file `calculator.rb`. Anyone with this file should be able to run the command `$ ruby calculator.rb` to run the program.
+
```
Comment (droberts-sea) [inline README.md:23]: You say "run" twice in this sentence.

File: README.md [hunk 2]
```diff
@@ -14,39 +13,70 @@
...
+- The program should accept both the name and the symbol for each possible operation. The expected names and symbols for each operation are the following:
+  - `add` and `+`
+  - `subtract` and `-`
+  - `multiply` and `*`
+  - `divide` and `/`
+- You may add other names or symbols to the above operations
+- The program should gracefully handle unexpected user input by doing the following things:
+  - check if the input is invalid. What happens if the user input is `nil` (i.e., the user just pressed enter)? What happens if the user tries to `add` `hotdog` to `elephant`? What if the user input is `hotdog` for an operator?
+  - If the input is invalid, output a message to the command line that informs the user that the input was invalid
+  - exit the program
```
Comment (droberts-sea) [inline README.md:45]: Capitalization

**Review by droberts-sea — APPROVED** [review]
Looks good

File: README.md [hunk 3]
```diff
@@ -14,39 +13,70 @@
...
+1. a number for the math operation
+1. a second number for the math operation
 
-## Primary Requirements
 The program should use the input operation and two numbers to provide the result of applying the operation to the two numbers.
+
 - The program should have support for these four operations: addition, subtraction, multiplication, and division.
-- The program should accept both the name (`add`) and the symbol (`+`) for each possible operation.
-- The program should handle unexpected user input
+- The program should accept both the name and the symbol for each possible operation. The expected names and symbols for each operation are the following:
```
Comment (shrutivanw) [inline README.md:35]: Question: Should the user input be considered case insensitive? Or is that not part of the program requirement? e.g. `add` versus `Add`.

**Review by shrutivanw — APPROVED** [review]
I like all the new changes. :+1:

**Pull request merged**